hwdb/60-evdev.hwdb

```diff
@@ -461,6 +461,13 @@ evdev:name:AlpsPS/2 ALPS DualPoint TouchPad:dmi:*svnTOSHIBA:pnTECRAM11*
  EVDEV_ABS_00=90:962:11
  EVDEV_ABS_01=51:681:14
 
+# Toshiba Satellite R830
+evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnTOSHIBA:pnSATELLITER830*
+ EVDEV_ABS_00=1238:5785:53
+ EVDEV_ABS_01=1045:4826:76
+ EVDEV_ABS_35=1238:5785:53
+ EVDEV_ABS_36=1045:4826:76
+ 
```

Comment:
The test seems to have failed because of an invisible space here. I'm not sure why it was impossible to reproduce locally. @whot I'm wondering what version of `pyparsing` you were using.

Reply:
I've just opened https://github.com/systemd/systemd/pull/8701.

Reply:
`python3-pyparsing-2.1.10-4.fc27.noarch`. Tested this again to make sure and `parse_hwdb.py` still succeeds. 

 ...  a bit later... 

doh. found the issue - when I applied the patch locally git stripped whitespace errors, so that one invisible space wasn't present locally. It does indeed fail with the upstream commit. Sorry about that.

Reply:
Not a problem. I'd actually say that the effect is rather positive because It turned out that `hwdb-parse.py` hadn't been run on Semaphore and Fedora CI, which has just been fixed. It would probably be great to tweak `.gitattributes` so that `git` wouldn't overwrite anything, but I'm not sure what exactly should be changed.

Reply:
I don't think anything can (or should) be done in systemd here, this is a local setting `apply.whitespace=fix` that I have in my gitconfig and it only takes effect if you `git am` or `git apply` the patch, which I did in this case.

Thanks heaps for fixing this btw, much appreciated.